quickwit/quickwit-serve/src/jaeger_api/rest_handler.rs

```diff
@@ -397,6 +407,7 @@ mod tests {
             .path(
                 "/otel-traces-v0_6/jaeger/api/traces?service=quickwit&\
                  operation=delete_splits_marked_for_deletion&minDuration=500us&maxDuration=1.2s&\
+                 tags=%7B%22tag.first%22%3A%22common%22%2C%22tag.second%22%3A%22true%22%7D&\
```

Comment:
we should check that the parameters are correctly built on the search request

Reply:
noted

Reply:
@fmassot is it something you are after?
```
.withf(|req|
                {
                    assert!(req.query_ast.contains("{\"type\":\"term\",\"field\":\"resource_attributes.tag.first\",\"value\":\"common\"}"));
                    assert!(req.query_ast.contains("{\"type\":\"term\",\"field\":\"resource_attributes.tag.second\",\"value\":\"true\"}"));
                    assert_eq!(req.index_id_patterns, vec![OTEL_TRACES_INDEX_ID.to_string()]);
                    true
                }
            )
```